api/autopilot.go

```diff
@@ -91,7 +91,12 @@ type (
 	// AutopilotStatusResponseGET is the response type for the /autopilot/status
 	// endpoint.
 	AutopilotStatusResponseGET struct {
-		CurrentPeriod uint64 `json:"currentPeriod"`
+		Configured           bool          `json:"configured"`
+		MigrationLastStarted ParamTime     `json:"migrationLastStarted"`
+		MigrationRunning     bool          `json:"migrationRunning"`
+		ScanLastStarted      ParamTime     `json:"scanLastStarted"`
+		ScanRunning          bool          `json:"scanRunning"`
+		Uptime               ParamDuration `json:"uptime"`
```

Comment:
Can you call that endpoint in a test to make sure `Configured` and the `ParamTime` and `ParamDuration` fields are set after starting a cluster?